Tests/PHPUnit/Services/Mollie/Payments/Converters/AddressConverterTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace MollieShopware\Tests\Services\Mollie\Payments\Builders;
+
+use MollieShopware\Services\Mollie\Payments\Converters\AddressConverter;
+use MollieShopware\Services\Mollie\Payments\Models\PaymentAddress;
+use PHPUnit\Framework\TestCase;
+
+class AddressConverterTest extends TestCase
```

Comment:
@boxblinkracer you're sure you want to use the actual address data instead of a fake one?

Reply:
yes, because this class is about testing that AddressConverter :)

Reply:
ok i have no clue what you mean.
i test the payment-to-array converter by providing a fixture payment and assert the resulting array?! :)